modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -137,6 +142,12 @@ template<typename Net> class Params {
         return *this;
     }
 
+    Params& setNumRequests(size_t nireq) {
+        GAPI_Assert(nireq > 0 && "Number of infer requests must be more than zero!");
```

Comment:
Simple test is required.

Reply:
more -> greater than (you're talking numbers)

Reply:
Fixed